sdcm/cluster.py

```diff
@@ -3227,7 +3305,9 @@ def get_rack_names_per_datacenter_and_rack_idx(self, db_nodes: list[BaseNode] |
 
         rack_names_mapping = {}
         for (region, rack), nodes in self.nodes_by_racks_idx_and_regions(nodes=actual_db_nodes).items():
-            rack_names_mapping[(region, rack)] = status[nodes[0]]['rack']
+            # If `nodetool status` failed to get status for the node
+            if rack_name := status.get(nodes[0]).get('rack'):
```

Comment:
same here

Reply:
fixed

Reply:
Same here. Where is it fixed? There are no code uploads since the comment was made.

Reply:
pushed